src/backends/torch/torchlib.cc

```diff
@@ -75,21 +74,6 @@ namespace dd
 #endif
   }
 
-  void add_parameters(std::shared_ptr<torch::jit::script::Module> module,
```

Comment:
The following functions are utility functions and should be moved in a separate header, e.g. `torchutil.h` so that they can be accessed by TorchModule, TorchSolver, etc

Reply:
done